enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -162,9 +160,29 @@ platform:
     hostedZone: Z00147933I3NWOQ6M4699
     hostedZoneRole: arn:aws:iam::<account-a>:role/<role-name>
 ```
-Please see Open Questions for further discussion of the install config.
 
+### API: External DNS Operator
+
+This enhancement also extends [External DNS Operator](https://github.com/openshift/external-dns-operator), an add-on
+operator deployable using Operator Lifecycle Manager (OLM), to support cross account DNS zones in AWS. We extend the
+`ExternalDNS` API object by adding the field `RoleARN` to `ExternalDNSAWSProviderOptions` struct (`spec.provider.aws`).
+Refer to [externaldns_types.go](https://github.com/openshift/external-dns-operator/blob/main/api/v1beta1/externaldns_types.go)
```

Comment:
Tiny copyedits:
```suggestion
operator deployable using Operator Lifecycle Manager (OLM), to support cross-account DNS zones in AWS. We extend the
`ExternalDNS` API object by adding the field `RoleARN` to `ExternalDNSAWSProviderOptions` struct (`spec.provider.aws`).
Refer to [`externaldns_types.go`](https://github.com/openshift/external-dns-operator/blob/main/api/v1beta1/externaldns_types.go)
```
I was going to suggest using a permalink, but in this case (pointing the the v1beta1 API, without any line number), I suppose it's appropriate to use the main branch reference.  

Consider adding a note that the proposed API change is purely additive and is therefore a non-breaking, backwards-compatible change.